app/menus/menu.js

```diff
@@ -12,7 +12,6 @@ const darwinMenu = require('./menus/darwin');
 module.exports = (createWindow, updatePlugins) => {
   const commands = getKeymaps().commands;
   const menu = [
-    (process.platform === 'darwin' ? darwinMenu(commands) : []),
```

Comment:
you copy pasted it wrong 😉 https://github.com/zeit/hyper/pull/1876#discussion_r118828269